### ISSUE TITLE
Update not working

### ISSUE THREAD

**Issue by keencave** [bubbledevteam/BubbleFW_FlashTool#1]
My update failed. After finding my Bubble device I started the upgrade. The bubble LED went red but the firmware download dont start. The app is waiting at 0% and reports after short time: Failed. After esetting the bubble via the magnet of the charging cable the LED went off and come back in red after reboot. 
Any ideas?
Should I use a JTAG adapter to download? Where can I get the current firmwafre binary?

**Comment by parapenT1sta** [comment]
Did you try Diabox app to update it?

**Comment by keencave** [comment]
Yes. Diabox finds the bubble device und try to upload. then it stuck when starting the upload at 0%. Red LED seems to indicate that bubble device is in OTA mode waiting for a new firmware?

**Comment by parapenT1sta** [comment]
I see, in that case maybe only nRF Toolbox can save you :-(

**Comment by keencave** [comment]
Fine. But where can I find the binary to upload?

**Comment by parapenT1sta** [comment]
Try to ask here if someone has it: https://www.facebook.com/groups/Bubblan.org/

**Comment by ghost** [comment]
Flashtool for fimware 2.0 should be updated soon.

**Comment by parapenT1sta** [comment]
The ZIP file is available here: https://github.com/bubbledevteam/BubbleFW_FlashTool/releases/tag/BubbleFirmware

**Comment by keencave** [comment]
Thx!
